### PR TITLE
Several changes:

### PR DESCRIPTION
**Pull request by avasquez614**
* Removed mariadb container. Studio will start its embedded MariaDB.
* Enhancements for backup/restore.
* Added more doc to README.md

### CODE REVIEW

**Comment by avasquez614** [comment]
@alhambrav can you check the doc in the REAME.md? Let me know if there's something wrong

File: README.md [hunk 1]
```diff
@@ -5,7 +5,10 @@ environments:
 
 * Authoring
 * Delivery
-* Serverless Delivery (content stored in S3)
+* Serverless Delivery (external ElasticSearch service and content stored in S3)
```
Comment (sumerjabri) [inline README.md]: Elasticsearch (small s)

File: README.md [hunk 2]
```diff
@@ -5,7 +5,10 @@ environments:
...
+
+**NOTE:** These files are intended for development use only. We recommend you create your own set of Docker compose
+files for production and use these ones as reference.
```
Comment (sumerjabri) [inline README.md]: ... and use these as a reference.

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sumerjabri) [reply README.md]: you're missing `a`

**Review by sumerjabri — CHANGES_REQUESTED** [review]
You're missing an 'a', see notes.

**Pull request merged**